source/includes/documentation/_material_ui_elements.md

```diff
@@ -3573,45 +3573,46 @@ Available methods in Java JDI Light:
 ### 4.46 Text Field
 
 ```java 
-    //    @FindBy(xpath = "//form[@id='formProps']/div[2]/div[contains(@class, 'MuiTextField-root')]")
-    @UI("//form[@id='formProps']/div[2]/div[contains(@class, 'MuiTextField-root')]")
+    //    @FindBy(xpath = "//*[@id='filled-error']/ancestor::*[1]")
+    @UI("//*[@id='filled-error']/ancestor::*[1]")
```

Comment:
the issue with this xpath is that it is now points on a different web element (see screenshot) which can lead to parsing issues for that web element
![image](https://user-images.githubusercontent.com/14041995/190124654-bb1f792e-ca85-4651-87c5-0e35d63bac41.png)
I suggest to use this one `//*[@id='filled-error']/ancestor::*[contains(@class, 'MuiTextField-root')]`